src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -81,6 +81,7 @@ export class LargeArtworkImageBrowser extends React.Component<
                 width="100%"
                 px={hasMultipleImages ? [2, 2, 0] : 0}
                 height="60vh"
+                maxHeight={image.height}
```

Comment:
@lilyfromseattle - this keeps coming up and I'm sorry for constantly flagging! This is the wrong approach as the image is supposed to scale. We can't lock it at a specific height because then it is not responsive. It's supposed to stretch at larger sizes. Totally down to jump on a call to explain this more in person and show exactly why.

Reply:
For now lets leave this out of this PR and handle it in its own instance.

Reply:
(will need to remove each of the other `maxHeights`)

@kierangillen - can you update the JIRA ticket with why, or close / retitle so that this isn't picked up again via this approach? I can also do this. Ideally, though, I can jump on a call with both of y'all and screen share show whats going on and why these various height solutions don't work for responsivity, independent of Google Bot.

Reply:
That height is the size of the image we're getting from the query. You're saying that the image is supposed to stretch beyond its actual dimensions?

Reply:
It seems like the image should only be scalable up to its actual dimensions, otherwise wont it pixelate?

Reply:
It will, and that's ok. For larger screens this is important, and depending on the aspect ratio it will stop at the edges.

Reply:
The way the page is laid out with `60vh` is deliberate, not a bug, and was carried over from the old Artwork page and discussed at length in the new artwork page rebuild. 

@kierangillen mentioned earlier that to fix the google bot screenshot we need to set an arbitrarily large maxHeight (say 2000px). When a bot visits that doesn't possess normal screen dimensions, it will cap there and be passible. There are possibly other solutions to this (using a media query for example), but capping a height won't work for our users. Happy to brainstorm (not sure how much priority this has) and can happy to go into more depths about this on a call.

Reply:
`image.height` is going to be around 2000px since we're fetching a large image. Is it better to hard code an arbitrary number?

Reply:
Is this always the case though? It depends on what has been entered into CMS. But if in general, based upon testing a bunch of different pages on a large apple monitor (not a laptop) things look as they currently do in prod, then 👍on this change.

Reply:
@lilyfromseattle - thoughts?

Reply:
I changed it to a hardcoded maxHeight

Reply:
I'm wondering about 

> But if in general, based upon testing a bunch of different pages on a large apple monitor (not a laptop) things look as they currently do in prod

Reply:
I tested on large external monitor, which is high resolution and the monitor apple recommends. It looks the same, as you can see in the screenshots above.

Reply:
Sweet, tyty 👍